docs/DeveloperGuide.md

```diff
@@ -360,6 +360,38 @@ This process is summarised in the activity diagram below
 
 ![Delete Activity Diagram](images/DeleteStudentActivityDiagram.png)
 
+#### Take Attendance Feature
+This section explains the implementation of the Take Attendance feature via the `takeattendance` command.
+The `TakeAttendanceCommand` causes the specified `Person` to be added to the specified session. 
```

Comment:
Might be good to also mention the parameters that the command will take in

Reply:
Sounds good!